lib/manageiq/rpm_build/rpm_repo.rb

```diff
@@ -0,0 +1,77 @@
+module ManageIQ
+  module RPMBuild
+    class RpmRepo
+      include S3Common
+
+      def update
+        require 'tempfile'
+        Dir.mktmpdir do |tmpdir|
+          work_dir = Pathname.new(tmpdir)
+
+          # Build directory structure /release/{master,jansa}/el8/{noarch,src,x86_64}
+          require 'fileutils'
+          directories = []
+          OPTIONS.rpm_repository.content.each do |branch, values|
+            values[:targets].each do |target|
+              OPTIONS.rpm_repository.arches.each do |arch|
+                directories << work_dir.join("release", branch.to_s, target, arch)
+              end
+            end
+          end
+          directories.each { |dir| FileUtils.mkdir_p(dir) }
+
+          # Fetch the configured RPMs
+          puts "Downloading required RPMs..."
+          require 'rest-client'
```

Comment:
I think we are trying to avoid rest-client since it is not being kept up to date.  I am ok for this PR, but we may want to move to just use Net:HTTP afterwards.